collibra.d.ts

```diff
@@ -149,5 +188,6 @@ declare namespace Collibra {
   export interface PagedAttributeResponse extends PagedResponse<Attribute> {}
```

Comment:
Open question for the team: should we have both interfaces and types in our codebase? In many projects I have seen only type to be used in order to have less confusion.

Reply:
I read an article once about what the difference is and when which should be used. essentially, the author of the article said that the way they read types and interfaces, was that interfaces are like classes—made to be extended. when they saw an interface, they expected it to get extended. meanwhile types were more static.

however, functionally they're the same. i also believe types don't accept generics.

Reply:
Yes, that is true. I just know that some developers get a bit confused on what to use where, so by taking one away the choice ise easier. To me it is more about distancing myself from object oriented programming :smile